src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,202 @@
+@media screen and (max-width: 1343px) {
```

Comment:
Please define some utility helper functions for the selection of the screen-sizes, so that can be reused in all other parts of the application

Reply:
Done ✅